tasks/deploy-hashed-assets.js

```diff
@@ -126,13 +124,16 @@ module.exports = function (program, utils) {
 	program
 		.command('deploy-hashed-assets')
 		.description('deploys hashed asset files to S3 (if AWS keys set correctly)')
-		.option('--monitor-assets', 'Will send asset sizes to Graphite')
-		.option('--directory <directory>', 'Directory to deploy (defaults to public)')
+		.option('--manifest-file <filename>', 'Name of the manifest file to read', defaultManifestFile)
+		.option('--directory <directory>', 'Directory containing the assets to deploy', defaultDirectory)
+		.option('--namespace <namespace>', 'Name of the folder to upload into', defaultNamespace)
```

Comment:
`--destination`? Not sure if that's better 🤔

Reply:
Maybe `--destination-directory` to disambiguate it from the bucket name?